scania-digital-design-system/tegel
# feat(prefix): changing prefix from sdds to tds

**Describe pull-request** 
Prefix change for out entire package.

- [x] Section in the migration docs
- [x] Proof read the migration docs
- [x] Typography
- [x] Grid
- [x] Mixins
- [x] Event docs
- [x] Badge
- [x] Banner
- [x] Block
- [x] Breadcrumbs
- [x] Card
- [x] Chip
- [x] Datetime
- [x] Divider
- [x] Footer
- [x] Link
- [x] Message
- [x] Popover Canvas
- [x] Popover Menu
- [x] Radio Button
- [x] Side Menu
- [x] Spinner
- [x] Stepper
- [x] Folder Tabs
- [x] Inline Tabs
- [x] Navigation Tabs
- [x] Textarea
- [x] Text Field
- [x] Toast
- [x] Toggle
- [x] Tooltip

**Solving issue**  
Fixes: #

**How to test**  
1. Go to storybook link below
2. Check in all stories
3. Make sure everything is working as expected.
